src/hooks/useJobs.tsx

```diff
@@ -25,5 +25,5 @@ export function useKillJobMutation() {
 }
 
 export function useJob(env: string, region: string, jobId: string) {
-  return useQuery<{ job: Job } | null, Error>(['jobId', jobId], () => fetchJob(env, region, jobId));
```

Comment:
I have since switched to not typing useQuery with types after it with angle brackets. I think the type system works better with inference in this case.

See: https://tkdodo.eu/blog/type-safe-react-query which is from the guy who now maintains the library.